bot.py

```diff
@@ -19,7 +19,7 @@ async def on_message(message):
         return 
 
     if "$shania" in message.content.lower():
-        await message.channel.send("Let's Go Girls")
+        await message.channel.send("Let's Go to Hell, Girls")
```

Comment:
Thanks for the PR. We're not interested in making Shania Evil right now, but we will look into this later around Halloween.